addons/l10n_ar_withholding/data/template/ir.sequence-ar_ri.csv

```diff
@@ -0,0 +1,27 @@
+id,implementation,name,prefix,code,padding,number_increment
+ri_seq_withholding_earnings_applied,standard,withholding Earnings applied,,,8,1
```

Comment:
This csv does not make any sense.  We know the information.

Reply:
@maq-adhoc is this ready to be closed? As I see this is Outdated.
cc' @jco-odoo

Reply:
DONE!